Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -832,3 +860,141 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Replicate Encrypted Properties wit
 }
 #endif //#ifdef COUCHBASE_ENTERPRISE
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - Remove Doc From Channel SG", "[.SyncServerCollection]") {
+    string idPrefix = timePrefix();
+    // one collection now now. Will use multiple collection when SG is ready.
+    constexpr size_t collectionCount = 1;
+    std::array<C4CollectionSpec, collectionCount> collectionSpecs = {
+        Roses
+    };
+    std::array<C4Collection*, collectionCount> collections =
+        collectionPreamble(collectionSpecs, "sguser", "password");
+    std::array<C4ReplicationCollection, collectionCount> replCollections;
+
+    string        doc1ID {idPrefix + "doc1"};
+    vector<string> chIDs {idPrefix+"a", idPrefix+"b"};
+
+    C4Error error;
+    DEFER {
+        REQUIRE(assignUserChannel(this, {"*"}, &error));
+    };
+    REQUIRE(assignUserChannel(this, chIDs, &error));
+
+    // Create docs on SG:
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl;
```

Comment:
Someone looking at this will have no idea what these credentials are supposed to be

Reply:
Let me change it to a constant.